packages/arb-token-bridge-ui/src/util/wagmi/setup.ts

```diff
@@ -26,29 +27,104 @@ const chainList = isTestingEnvironment
     ]
   : [mainnet, arbitrum, arbitrumNova, goerli, arbitrumGoerli]
 
-const { chains, provider } = configureChains(chainList, [publicProvider()])
+const projectId = process.env.NEXT_PUBLIC_WALLETCONNECT_PROJECT_ID!
 
 const appInfo = {
-  appName: 'Bridge to Arbitrum'
+  appName: 'Bridge to Arbitrum',
+  projectId
 }
 
-const { wallets } = getDefaultWallets({
-  ...appInfo,
-  chains
-})
+const TargetChainKeys = [
```

Comment:
NIT: We could use enum here instead:
```
enum TargetChainKeys {
  'mainnet' = 'mainnet',
  'arbitrum-one' = 'arbitrum-one',
  'arbitrum-nova' = 'arbitrum-nova',
  'goerli' = 'goerli',
  'arbitrum-goerli' = 'arbitrum-goerli'
}

function sanitizeTargetChainKey(targetChainkey: string | null): TargetChainKeys {
  if (targetChainkey === null) {
    return TargetChainKeys.mainnet
  }

  if (!(targetChainkey in TargetChainKeys)) {
    return TargetChainKeys.mainnet
  }

  return targetChainkey as TargetChainKeys
}
```

Then we also don't need `TargetChainKey` type anymore